hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -458,7 +458,7 @@ public final void readData(ObjectDataInput in) throws IOException {
     @Override
     public String toString() {
         final StringBuilder sb = new StringBuilder(getClass().getName()).append('{');
-        sb.append("serviceName='").append(serviceName).append('\'');
+        sb.append("serviceName='").append(getServiceName()).append('\'');
```

Comment:
`serviceName` is null by default and it's useless to print null here. Most of the operations are overriding `getServiceName()` to return a hardcoded service-name.